src/main/java/de/hpi/bpt/argos/persistence/database/DatabaseConnectionImpl.java

```diff
@@ -213,60 +171,43 @@ public Product getProduct(long productId) {
 	@Override
 	public EventType getEventType(long eventTypeId) {
 		Session session = databaseSessionFactory.openSession();
-		Transaction tx = null;
-		Query<EventType> query = null;
-		try {
-			tx = session.beginTransaction();
+		Transaction transaction = session.beginTransaction();
+		Query<EventType> query = session.createQuery("FROM EventTypeImpl et "
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FROM EventTypeImpl et " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)